src/PKSim.Assets/PKSimConstants.cs

```diff
@@ -2157,6 +2160,7 @@ public static class UI
          public static readonly string UseWatermark = "Use watermark in charts when exporting to clipboard?";
          public static readonly string WatermarkText = "Text";
          public static readonly string WatermarkProperties = "Watermark Properties";
+         public static readonly string CreateTableSolubilityAlternative = "Create as pH-Sol table";
 
```

Comment:
Think _Create as pH-Sol**ubility** table_ would be better

Reply:
:)
![image](https://user-images.githubusercontent.com/1041237/38932048-1a508690-42e3-11e8-9c18-8c91f55558e9.png)

changed

Reply:
done